endpoints/setuid.go

```diff
@@ -110,6 +110,10 @@ func NewSetUIDEndpoint(cfg *config.Configuration, syncersByBidder map[string]use
 			return
 		}
 
+		// bidderNameBySyncerKey will be used here for activities integration.
+		// usage:
+		// bidderName := bidderNameBySyncerKey[syncer.Key()]
```

Comment:
We can't do this. There is a 1:many relationship between a syncer key and the bidder name.

Instead, we need to change the callback/redirect url to include the bidder name instead of the syncer key. The setUID endpoint would then need to use an existing map of bidder name to syncer to get the key for the cookie map.

Reply:
Right, this is an array, more like `bidderNames`. 
Oh ok, Let me see what I can do...